__tests__/main.test.ts

```diff
@@ -135,6 +136,25 @@ describe('validate', () => {
       expect(() => getSetCommitsOption()).toThrow(errorMessage);
     });
   });
+  describe('getProjects', () => {
+    afterEach(() => {
+      delete process.env['SENTRY_PROJECT'];
+      delete process.env['INPUT_PROJECTS'];
+    });
+    it('read from env variable', () => {
+      process.env['SENTRY_PROJECT'] = 'my-proj';
+      expect(getProjects()).toEqual(['my-proj']);
+    });
+    it('read from option', () => {
+      process.env['INPUT_PROJECTS'] = 'my-proj1 my-proj2';
+      expect(getProjects()).toEqual(['my-proj1', 'my-proj2']);
+    });
+    it('throws error if no project', () => {
```

Comment:
Can you add a test where you set both `SENTRY_PROJECT` and `INPUT_PROJECTS` to show that one overwrites the other.

Reply:
@mgaeta added